lib/conventions.js

```diff
@@ -55,15 +53,13 @@ function freeze(object, oc) {
 }
 
 /**
- * Since xmldom can not rely on `Object.assign`,
- * it uses/provides a simplified version that is sufficient for its needs.
+ * Since xmldom can not rely on `Object.assign`, it uses/provides a simplified version that is sufficient for its needs.
  *
  * @param {Object} target
  * @param {Object | null | undefined} source
- *
- * @returns {Object} target
- * @throws TypeError if target is not an object
- *
+ * @returns {Object} Target.
+ * @throws
+ * TypeError if target is not an object.
```

Comment:
This happens for two reasons:

1. I believe the columns were disabled, so the format is now `@tag {type}\ndescription`
2. The type is not between curly brackets:

```
@throws free-form description
@throws {<type>}
@throws {<type>} free-form description
```
> [`@throws` doc](https://jsdoc.app/tags-throws.html)

So, this should be `@throws {TypeError} If target is not an object.`

Reply:
This is something that bothers me as well. It seems there is no way to do this. As soon as columns are disabled, it forces all descriptions to be on newlines.

Reply:
@homer0 
`@throws {<type>} free-form description` does not work either when columns are disabled. It will always put description in new line.

Reply:
Sorry, no, I didn't mean to say that you could do it "fix it" by changing the syntax; What I meant to say is that if you were to use `@throws {TypeError} If target is not an object.`, the type would stay next to the `@throws` instead of just moving everything to the next line:

```
 * @throws {TypeError}
 * If target is not an object.
```

Sorry for the confusion.

Reply:
@shunkica yeah, it doesn't seem like it would be too complicated: maybe two new options, to specify that the mode is enabled, and, the indentation value for the next lines.

Unfortunately, my current issue is time :P; I only have a few hours at night to work on personal stuff. I'll take note and add it to my "backlog", but I can't promise you that I'll make it a priority, sorry.